api/numaresourcesoperator/v1/numaresourcesoperator_types.go

```diff
@@ -141,10 +140,6 @@ type NUMAResourcesOperatorStatus struct {
 type MachineConfigPool struct {
 	// Name the name of the machine config pool
 	Name string `json:"name"`
-	// Conditions represents the latest available observations of MachineConfigPool current state.
```

Comment:
we can't remove fields from the API until we bump the version. The best we can do is not set them (which we already do)